logprep/processor/concatenator/processor.py

```diff
@@ -20,7 +20,7 @@
             - tests/testdata/rules/generic/
 """
 from logprep.abc.processor import Processor
-from logprep.processor.base.exceptions import DuplicationError
+from logprep.processor.base.exceptions import FieldExsistsWarning
 from logprep.processor.concatenator.rule import ConcatenatorRule
 from logprep.util.helper import add_field_to, get_dotted_field_value
```

Comment:
Seems like you are sorting imports now, but are not yet automatically removing unused imports :wink: 
```suggestion
from logprep.processor.concatenator.rule import ConcatenatorRule
from logprep.util.helper import get_dotted_field_value
```